cli/src/java/org/apache/hadoop/hive/cli/CliDriver.java

```diff
@@ -445,6 +445,17 @@ public void handle(Signal signal) {
         }
         break;
       case '"':
+        if (!escape) {
```

Comment:
I'm pretty sure the Hive maintainers will want unit tests that cover all of these cases.

Reply:
> I'm pretty sure the Hive maintainers will want unit tests that cover all of these cases. See https://github.com/apache/hive/blob/master/cli/src/test/org/apache/hadoop/hive/cli/TestCliDriverMethods.java

I hava added the unit test for these cases, when  a semicolon in double quote.